File: client/src/app/_models/hr/interview.ts
import {IInterviewItem} from "./interviewItem";

export interface IInterview{
     id: number;
     orderId: number;
     orderNo: number;
     orderDate: Date;
     customerId: number;
     customerName: string;
     interviewMode: string;
     interviewDateFrom: Date;
     interviewDateUpto: Date;
     interviewStatus: string;
     interviewItems: IInterviewItem[];
}

File: api/Entities/Admin/Interview.cs
namespace api.Entities.Admin
{
    public class Interview: BaseEntity
    {
        public int OrderId { get; set; }
        public int OrderNo { get; set; }
        public DateTime OrderDate { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public DateTime InterviewDateFrom { get; set; }
        public DateTime InterviewDateUpto { get; set; }
        public string InterviewStatus { get; set; }
        public ICollection<InterviewItem> InterviewItems { get; set; }
    }
}